google/cloud/spanner/mocks/mock_database_admin_connection.h

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_SPANNER_GOOGLE_CLOUD_SPANNER_MOCKS_MOCK_DATABASE_ADMIN_CONNECTION_H_
+#define GOOGLE_CLOUD_CPP_SPANNER_GOOGLE_CLOUD_SPANNER_MOCKS_MOCK_DATABASE_ADMIN_CONNECTION_H_
+
+#include "google/cloud/spanner/database_admin_connection.h"
+#include <gmock/gmock.h>
+
+namespace google {
+namespace cloud {
+namespace spanner_mocks {
+inline namespace SPANNER_CLIENT_NS {
+/**
```

Comment:
nit: I think most of our files have a single blank line after opening this inline namespace. I think clang-format used to remove this blank line, then I it stopped removing it, not sure what it's doing now :-)

Reply:
Fixed.  I think clang-format preserves 1, but no more, blank lines here.